model/client4.go

```diff
@@ -212,6 +212,17 @@ func (c *Client4) UpdateUser(user *User) (*User, *Response) {
 	}
 }
 
+// UpdateUserPassword updates a user's password.
+func (c *Client4) UpdateUserPassword(userId, currentPassword, newPassword string) (*User, *Response) {
+	requestBody := map[string]string{"current_password": currentPassword, "new_password": newPassword}
+	if r, err := c.DoApiPut(c.GetUserRoute(userId)+"/password", MapToJson(requestBody)); err != nil {
+		return nil, &Response{StatusCode: r.StatusCode, Error: err}
+	} else {
+		defer closeBody(r)
+		return UserFromJson(r.Body), BuildResponse(r)
```

Comment:
im I reading this wrong? the API says that it returns `ReturnStatusOK(w)` but the driver returns a `*User`?